source/upgrade/important-upgrade-notes.rst

```diff
@@ -3,12 +3,22 @@ Important Upgrade Notes
 
 .. important::
    - Support for Mattermost Server v5.31 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ will come to the end of its life cycle on October 15, 2021. Upgrading to Mattermost Server v5.37 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ or later is required.
-   - In the v5.38 release (August 16, 2021), we will deprecate "config watcher" (the mechanism that automatically reloads the ``config.json file``), in favor of an mmctl command that will need to be run to apply configuration changes after they are made. This change will improve configuration performance and robustness.
    - The deprecations `listed here <https://docs.mattermost.com/administration/changelog.html#upcoming-deprecations-in-mattermost-v6-0>`_ are planned for the Mattermost v6.0 release, which is scheduled for September 15, 2021. This list is subject to change prior to the release.
+   - Upgrading Mattermost can result in an error “ERROR: default for column "timezone" cannot be cast automatically to type jsonb” if you have non-JSON data in a column. This can also happen if the database data has been manipulated by external processes (e.g. plugins) and in cases where improper input data could get silently truncated due to varchar limits. The affected database column has to be manually fixed.
```

Comment:
Looks good. I'd just replace "timezone" with a generic "column_name" since it's not something specific to that column but could happen on any of the affected columns which are listed in the release note for that change.

Also I'd change the last sentence into:

`The data in the affected database column has to be manually fixed.`